commandhandler/aggregate/commandhandler.go

```diff
@@ -72,3 +65,20 @@ func (h *CommandHandler) HandleCommand(ctx context.Context, cmd eh.Command) erro
 
 	return h.store.Save(ctx, a)
 }
+
+// HandleCommand handles a command with the registered aggregate.
+// Returns ErrAggregateNotFound if no aggregate could be found.
+func (h *CommandHandler) HandleCommand(ctx context.Context, cmd eh.Command) error {
+	err := eh.CheckCommand(cmd)
+	if err != nil {
+		return err
+	}
+
+	for {
```

Comment:
Not sure why this is needed, what was the reason behind it?

Reply:
I think he wants to retry a command if `ErrConcurrentException` happen, but It should live outside of Aggregate. That behavide was not expected.